### PR TITLE
Remove rinkeby references

### PR DESCRIPTION
**Pull request by ivanzhelyazkov**
This PR removes all Rinkeby references from origin-dollar.
Rinkeby is not currently used for staging or stable, and there is no plan to support a testnet in the near future, since there are no good mocks for most of the protocols which OUSD interacts with.
References issue:
https://github.com/OriginProtocol/origin-dollar/issues/1080

### CODE REVIEW

File: dapp/src/utils/web3.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 const networkInfo = {
   1: 'Mainnet',
   3: 'Ropsten',
```
Comment (franckc) [inline dapp/src/utils/web3.js:3]: Let's also cleanup those other testnet and only keep Mainnet and Localhost

File: dapp/src/utils/web3.js [hunk 2]
```diff
@@ -26,7 +25,6 @@ export function getEtherscanHost(web3React) {
   const chainIdToEtherscan = {
     1: 'https://etherscan.io',
     3: 'https://ropsten.etherscan.io',
```
Comment (franckc) [inline dapp/src/utils/web3.js:27]: let's clean up ropsten since it is not used

**Review by franckc — COMMENTED** [review]
Nice! Great cleanup. Couple small comments inline.

**Comment by franckc** [comment]
@sparrowDom I took a pass but could you please also review this PR? Ty ser!

File: dapp/src/utils/web3.js (hunk 1 above)
Comment (ivanzhelyazkov) [reply dapp/src/utils/web3.js]: Removed in the latest commit

File: dapp/src/utils/web3.js (hunk 2 above)
Comment (ivanzhelyazkov) [reply dapp/src/utils/web3.js]: Removed in the latest commit

**Review by shahthepro — APPROVED** [review]
LGTM

**Comment by franckc** [comment]
@sparrowDom Can you take a final look at this PR and then merge? ty :)

File: dapp/src/utils/web3.js [hunk 3]
```diff
@@ -1,9 +1,5 @@
 const networkInfo = {
   1: 'Mainnet',
-  3: 'Ropsten',
```
Comment (sparrowDom) [inline dapp/src/utils/web3.js:3]: I think we can keep this in. From what I remember these map nework IDS to names. Could still be useful for the dapp to be able to tell the name of the network when a user switches to a network the dapp does not support.

**Review by sparrowDom — APPROVED** [review]
Sweet thanks for doing this massive cleanup 👏 . One minor comment inline

File: dapp/src/utils/web3.js (hunk 3 above)
Comment (sparrowDom) [reply dapp/src/utils/web3.js:3]: Have checked the code, currently we fetch the name of the network only when successfully connected. So for current functionality of the dapp it is ok that they stay removed.

**Pull request merged**